nodestream/pipeline/extractors/queues/sqs.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+from logging import getLogger
+from typing import Any, AsyncGenerator, Iterable, Optional
+
+from nodestream.pipeline.extractors.queues.extractor import QueueConnector
+
+from ..credential_utils import AwsClientFactory
+
+
+class SQSQueueConnector(QueueConnector, alias="sqs"):
+    @classmethod
+    def from_file_data(
+        cls,
+        queue_url: str,
+        message_system_attribute_names: Optional[list[str]] = ["All"],
+        message_attribute_names: Optional[list[str]] = ["All"],
+        max_batch_size: int = 10,
+        delete_after_read: bool = True,
+        max_batches: int = 10,
+        **aws_client_args,
+    ):
+        return cls(
```

Comment:
It is possible to add a test case for this? It looks like it may be tricky and that may have been why you omitted it in which case I am okay.